cetloxblock/cetloxblock/cetloxblock.py

```diff
@@ -0,0 +1,165 @@
+# -*- coding: utf-8 -*-
+""" 
+This XBlock displays a CET LO item.
+The studio view enables the author to select the LO for the current xblock instance.
+"""
+
+import pkg_resources
+#import urllib
+from xblock.core import XBlock
+#from xblock.fields import String, Scope, ScopeBase, UserScope, BlockScope
+from xblock.fields import String, Scope
+from xblock.fragment import Fragment
+
+class CetLoXBlock(XBlock):
+  """ 
+  This XBlock displays a CET LO item.
+  The studio view enables the author to select the LO for the current xblock instance.
+  """
+
+  CET_DOMAIN = u".lab.cet.ac.il"
+  ITEM_URL_TEMPLATE = u"//lo{cetdomain}/player/?document={documentid}&language={language}&sitekey=ebag#options=nobar"
+  TASK_URL_TEMPLATE = u"//lo{cetdomain}/player/xblockplayer.html?task={taskid}&sitekey=ebag"
+  #TASK_URL_TEMPLATE = u"//lo{cetdomain}/player/?task={taskid}&sitekey=ebag#options=nobar"
+
+  #############################################################################
+  # ALL STRINGS ARE UNICODE!
+  # ALL FUNCTION DEFS GET UNICODE PARAMETERS AND RETURN UNICODE VALUES!
+  # KEEP THIS SO FOR YOUR HEALTH AND WELL BEING.
+  #############################################################################
+
+  # EDX field: Component's display name in studio and course
+  display_name = String(display_name="Display Name",
+    help="This name appears in the horizontal navigation at the top of the page.",
+    scope=Scope.settings,
+    default=u"CET LO Activity"
+  )
+
+  documentid = String(default="", scope=Scope.content, help="The LO id")
+  language = String(default="he", scope=Scope.content, help="The LO language code")
+  taskid = String(default="", scope=Scope.user_state, help="The LO id")
+
+  def resource_string(self, path):
+    """ helper for getting UNICODE STRINGS from static resources (from readthedocs/xblock tutorial)."""
+    data = pkg_resources.resource_string(__name__, path)
+    data = data.decode("utf8")
+    return data
+
+  has_author_view = True

```

Comment:
It would probably be better to move this up somewhere, so it's not in between functions, to keep things organized.